### ISSUE TITLE
The loader for the POCO X3 PRO is not working 

### ISSUE THREAD

**Issue by 0v3rfl00w** [bkerler/Loaders#105]
![Screenshot 2024-11-07 162033](https://github.com/user-attachments/assets/ed0d019a-1407-414c-8ebf-f53892695bd2)
Does anyone have an idea what is the problem here ?

**Comment by RenateUSB** [comment]
It could be SBL version. Can you get the SBL version? Do you have the xbl for this thing?

**Comment by 0v3rfl00w** [comment]
> It could be SBL version. Can you get the SBL version? Do you have the xbl for this thing?

Serial : 0x210a9b24 
  SoC : [ Unknown ] [ Snapdragon ]
  OEM : [ Qualcomm ] [ 0xMSM_HW_ID: 0x000a50e1 ] 
  PK_HASH[0] : db4b01086063007334de9e5ca14971c7c4f4358ec9d79cda46
  PK_HASH[1] : 1bebe3863a6781db4b01086063007334de9e5ca14971c7c 
  **SBL_SW :  0x00000002**

**Comment by RenateUSB** [comment]
Ya know... It's a lot easier to copy from **text** than from a **graphic**. Sigh.

If I have this correct, the Firehose loader you are using is SBL 1 and won't work.

**Comment by 0v3rfl00w** [comment]
> Ya know... It's a lot easier to copy from **text** than from a **graphic**. Sigh.
> 
> If I have this correct, the Firehose loader you are using is SBL 1 and won't work.

I thought it wasn't working because of the authentication required by Xiaomi , 
Is it possible to find a working firehose file

**Comment by 0v3rfl00w** [comment]
that's right , working with a loader from the stock firmware gives an EDL authentification error.
i dont know if bypassing this limitation is possible for this model

**Comment by chiteroman** [comment]
Using firehoses from stock and eng rom give the same error. I think we need to pay to repair this phone :/
